mysqld_exporter.go

```diff
@@ -726,6 +729,14 @@ func NewExporter(dsn string) *Exporter {
 			Name:      "scrapes_total",
 			Help:      "Total number of times MySQL was scraped for metrics.",
 		}),
+		scrapeErrors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: exporter,
+			Name:      "scrape_errors_total",
+			Help:      "Total number of times an error occured scraping a MySQL.",
+		},
```

Comment:
This is syntactically inconsistent with how we do it elsewhere.
Let's move the labels and the closing `)` directly into this line.

Also `scrapeErrorLabels` is not used elsewhere and pretty simple. We can just directly inline it here – makes it easier to read.